source/context.cpp

```diff
@@ -360,8 +360,8 @@ PYBIND11_MODULE({1}, root_module) {{
 
 
 const char *module_header = R"_(
-#include <pybind11/pybind11.h>
 #include <functional>
+#include <pybind11/pybind11.h>
```

Comment:
hm... it is odd that these include blocks got rearranged, - i guess different clang-format versions might have some discrepancies...

Reply:
It's good to have them rearranged in standard form - even if clang13 is wrong about touching code in quotes. Assuming that this is corrected in clang14, I would expect that clang14 won't revert these changes. Can you confirm?